src/common/utils/account.ts

```diff
@@ -20,6 +20,9 @@ export class AccountUtils {
     let account: Account;
     if (account_name) {
       account = (await api.get(`/accounts/${account_name}`)).data;
+      if (!account) {
+        throw new Error(`Could not find account=${account_name}`);
```

Comment:
Is it possible for no account to be returned? I would have assumed the api would throw a 404

Reply:
Yes, potentially unnecessary. I added it to cover all bases to guarantee that this method returned an account or threw (so its callers don't have to check if `account` is defined).

But if you think we can rely on the API for that in this case I can remove.